Dockerfile.release

```diff
@@ -1,23 +1,31 @@
-FROM bitwalker/alpine-erlang:19.2.1b
+FROM phusion/baseimage
```

Comment:
Why the change to phusion/baseimage? To be aligned with MongooseIM's docker image?